english/batman/batman.md

```diff
@@ -23,7 +23,7 @@ this realization, then you go:
 
 Bruce Wayne: It's not who I am underneath, but what I do that defines me.
 
-Commissioner Gordon: He's the hero Gotham deserves but not the one that it needs right now.
+[Commissioner Gordon](http://batman.wikia.com/wiki/James_Gordon): He's the hero Gotham deserves but not the one that it needs right now.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.